examples/niscope/graph-measurement-betterproto.py

```diff
@@ -119,7 +123,7 @@ async def ConfigureGrpcScope(scope_service: niscope_grpc.NiScopeStub, channel, v
     set_trigger_result = await scope_service.set_attribute_vi_int32(
         vi = vi,
         attribute_id = niscope_grpc.NiScopeAttributes.NISCOPE_ATTRIBUTE_TRIGGER_TYPE,
-        value = niscope_grpc.NiScopeInt32AttributeValues.NISCOPE_INT32_TRIGGER_TYPE_VAL_EDGE_TRIGGER
+        value_raw = niscope_grpc.NiScopeInt32AttributeValues.NISCOPE_INT32_TRIGGER_TYPE_VAL_EDGE_TRIGGER
```

Comment:
@harsha-bhushan  we have to set `value_raw` instead of `value` for any of the `oneof` fields in `.proto` file because of following bug in `betterproto`
> Note: The Better Protobuf compiler has a bug generating helpers for gRPC messages with oneof types. The problem is that zero-value messages in a oneof group take priority based on order and overwrite other set values. Therefore, only the last field, the field that has '_raw` appended to the name, can be properly set without wrapper modification.

We have already documented this in our [Creating-a-gRPC-Client](https://github.com/ni/grpc-device/wiki/) wiki. Should we add a comment here to similar effect so that users referring to betterproto example are made aware of it.

Reply:
Also, do you think this bug should be cherry-picked to releases\1.1 branch? Should we hold off release for this?

Reply:
If we just cherry pick to the release branch, the released client will not have the fix. Since example was working without error in previous release, and our changes have caused the bug, I would prefer that atleast just the bug fix part to get submitted by tomorrow before creating release. @ni-pkhare , would it be possible to separate just bug fix part to another tiny PR and we can ask owners to review it today since we need that for release? This PR of handling exception is fine to be just cherry picked to release branch

Reply:
I don't think size of change matters for cherry picking, we need to get change in main and create cherrypick PR after that to submit it in releases branch. Question is can we delay release by another day for that.

Reply:
1. A one-liner educating the users to use raw value should be good. We should avoid having a big paragraph.
2. Since we are anyway waiting on the NI-Sync team for updating release notes, I don't have a concern waiting for another day. My recommendation - fix the functionality bug as part of this PR & get it into the release.
The consistency related changes from other comments is not that critical. We could also do the clean up in future releases when we provide more better proto examples & discover better coding patterns.

Reply:
I think, this conversation can be resolved as it has been addressed seperately.